src/main/java/seedu/address/model/policy/Coverage.java

```diff
@@ -20,6 +20,7 @@
 
     public static final String MESSAGE_CONSTRAINTS =
             "Coverage should only contain alphanumeric characters and spaces, and it should not be blank";
+    public static final String DATA_TYPE = "COVERAGE";
```

Comment:
Bug: `addPolicy` with invalid coverage allows addition of policy
Steps to reproduce bug:
`addpolicy n/Senior Care d/Insurance for elderly c/days/5 months/14 p/$50000 sa/50 ea/75 `